test/smoke_test/utils.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+function check_exit_code {
```

Comment:
we're inventing pytest again here, don't we?

Reply:
discussed internally